src/index.css

```diff
@@ -296,3 +302,70 @@ div#picker {
     width: 100%;
   }
 }
+.logsignin{
```

Comment:
provide a comment before the style so anyone can understand which section is the css for. Also logsignin is not a proper naming convention. Use proper classnames, for example refer to the other classnames or [This article](https://www.freecodecamp.org/news/css-naming-conventions-that-will-save-you-hours-of-debugging-35cea737d849/)

Reply:
this applies to the other classname as well below.